src/request/remove-key/remove-key.css

```diff
@@ -64,3 +57,31 @@
 .page-footer {
     padding: 1rem;
 }
+
+.page#remove-key .backup-option {
+    display: flex;
+    align-items: center;
+    margin-top: 2rem;
+    max-width: 84%;
+}
+
+.page#remove-key .backup-option .login-file-svg-container {
+    min-width: 90px;
```

Comment:
Generally I would prefer `rem` over `px` in order for them to shrink with the rest of the content. These are also only `~67px` so around `8.375rem`